packages/core/src/types.ts

```diff
@@ -1581,6 +1587,7 @@ export interface StateConfig<
   done?: boolean;
   tags?: Set<string>;
   machine?: StateMachine<TContext, TEvent, any, any, any>;
+  _internalQueue?: Array<SCXML.Event<TEvent>>;
```

Comment:
Only used to queue up an `error.execution` event if the `microstep` procedure fails